portals/publisher/source/src/app/components/Apis/Details/Policies/GatewaySelector.tsx

```diff
@@ -145,7 +151,7 @@ const GatewaySelector: FC<GatewaySelectorProps> = ({
                                                     apiFromContext,
                                                 )}
```

Comment:
Line 138 aria attribute should be handled in [this way](https://v4.mui.com/components/radio-buttons/#accessibility)

Reply:
Fixed with: https://github.com/wso2/apim-apps/pull/211/commits/d6d0f77285306bbc17f0e0ccc35006a166d5429f